definitions/v3/blutopia-api.yml

```diff
@@ -54,10 +54,19 @@ search:
   paths:
     # https://hdinnovations.github.io/UNIT3D-Community-Edition-Docs/api_endpoints.html
     # https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/master/app/Http/Controllers/API/TorrentController.php
-    - path: "/api/torrents/filter?api_token={{ .Config.apikey }}&name={{ if .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}{{ if .Query.TMDBID }}&tmdbId={{ .Query.TMDBID }}{{ else }}{{ end }}{{ if .Query.IMDBIDShort }}&imdbId={{ .Query.IMDBIDShort }}{{ else }}{{ end }}{{ if  .Query.TVDBID }}&tvdbId={{ .Query.TVDBID }}{{ else }}{{ end }}&sortField={{ .Config.sort }}&sortDirection={{ .Config.type }}&perPage=100&page=1{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
+    - path: "/api/torrents/filter?sortField={{ .Config.sort }}&sortDirection={{ .Config.type }}&perPage=100&page=1{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
       response:
         type: json
         attribute: attributes
+  inputs:
+  # if we have an id based search, add Season and Episode as query in name for UNIT3D < v6.  Else pass S/E Params for UNIT3D >= v6
+    api_token: "{{ .Config.apikey }}"
+    name: "{{ if or (.Query.TVDBID) (.Query.IMDBID) (.Query.TMDBID) }}{{ if .Query.Season }}S{{ .Query.Season }}{{else}}{{ end }}{{ if .Query.Ep }}E{{ .Query.Ep }}{{else}}{{ end }}{{else}}{{ .Keywords }}{{ end }}"
+    tmdbid: "{{ if .Query.TMDBID }}{{ .Query.TMDBID }}{{else}}{{ end }}"
+    tvdbid: "{{ if .Query.TVDBID }}{{ .Query.TVDBID }}{{else}}{{ end }}"
+    imdbid: "{{ if .Query.IMDBIDShort }}{{ .Query.IMDBIDShort }}{{else}}{{ end }}"
+    # Supported with UNIT3d v6 only season: "{{ if .Query.Season }}{{ .Query.Season }}{{else}}{{ end }}"
+    # Supported with UNIT3d v6 only ep: "{{ if .Query.Ep }}{{ .Query.Ep }}{{else}}{{ end }}"
```

Comment:
Does the v6 beta not support season and episode searching, just the final v6?

Reply:
Vinnie just made the commit to support this the other week and not sure who is running that beta or newer other than blutopia at this point